src/components/cylc/GraphNode.vue

```diff
@@ -77,23 +60,10 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
           scale(${ (index === 0) ? mostRecentJobScale : '1' })
         `"
       >
-        <symbol
-          :id="`${nodeID}-${index}`"
+        <job
```

Comment:
Does this need a change or can I merge this one in?

Reply:
I was just waiting to see if there were any other suggestions before applying them in a batch (I try to avoid cluttering things with "address feedback" commits).